test/stylesheet-registry.js

```diff
@@ -104,6 +104,26 @@ test('add - sanitizes dynamic CSS on the server', t => {
   ])
 })
 
+test('add - nonce is properly fetched from meta tag', t => {
+  // We need to stub a document in order to simulate the meta tag
+  global.document = {
```

Comment:
now that I look at this again I realize that we should restore `global.document` after the test:

```js
const originalDocument = global.document

global.document = { /* ... */ }

// test ...

global.document = originalDocument
```

Reply:
It's undefined in the test environment as far as I could tell.

Reply:
yea but better reset in case something else relies on it to not be defined (good practice)